server.py

```diff
@@ -3,36 +3,49 @@
 from time import sleep
 from time import time
 class ClientChannel(PodSixNet.Channel.Channel):
-    def Network(self, data):
+    '''client channel class docstring to do'''
+
+    def Network(self, data: dict):  # Function name isn't very descriptive!
+        '''Console log. Prints data dictionary'''
+
         print(data)
-    def Network_readyStatus(self, data):
+
+    def Network_readyStatus(self, data: dict):
+        '''Feeds data into PodSixNet server object?'''
+
         self._server.readyStatus(data)
-    def Network_recieveCommand(self, data):
-        self._server.start_time = time()
+
+    def Network_recieveCommand(self, data: dict):
+        '''Pushes data into PodSixNet server objects'''
+
+        self._server.start_time = time()    #    testing latency
         self._server.recieveCommand(data)
 
-    #ping stuff
+    ####    ping stuff  ####
```

Comment:
Made the demarcation between "ping stuff" and the rest of the code stand out more.